app/scripts/templates/why_connect_another_device.mustache

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Yeah, that's how it is in almost all the other templates after the `header` too.